src/App.tsx

```diff
@@ -15,10 +48,14 @@ export const App: React.FC = () => {
 
       <div className="todoapp__content">
         <header className="todoapp__header">
-          {/* this buttons is active only if there are some active todos */}
-          <button type="button" className="todoapp__toggle-all active" />
+          <button
```

Comment:
jsx looks fine, but lets try to split it into separate components - for example header, todoList and footer (feel free to come up with your own component names)